tfhe/src/integer/ciphertext/mod.rs

```diff
@@ -76,8 +89,33 @@ impl RadixCiphertext {
     }
 }
 
-impl From<CompressedRadixCiphertext> for RadixCiphertext {
-    fn from(compressed: CompressedRadixCiphertext) -> Self {
+/// Structure containing a ciphertext in radix decomposition
+/// holding a signed value.
+#[derive(Serialize, Clone, Deserialize, PartialEq, Eq, Debug)]
+pub struct BaseSignedRadixCiphertext<Block> {
```

Comment:
Should we have a BaseUnsignedRadixCiphertext or an alias and RadixCiphertext would become something dynamic over the signedness ?

Not saying it's a good idea, thinking out loud

Reply:
Yeah I thought about renaming RadixCiphertext to UnsignedRadixCiphertext, I think it can be a good idea but needs more inputs

Reply:
more inputs ?

Reply:
more people 'voting' on whether we can rename RadixCiphertext to UnsignedRadixCiphertext

Reply:
agreed